salt/metalk8s/kubeadm/init/kubelet-start/files/service-kubelet-systemd.conf

```diff
@@ -0,0 +1,10 @@
+# Note: This dropin only works with kubelet v1.11+
+[Service]
+Environment="KUBELET_CONFIG_ARGS=--config={{ conf_file }}"
+# This is a file that "kubeadm init" and "kubeadm join" generates at runtime, populating the KUBELET_KUBEADM_ARGS variable dynamically
+EnvironmentFile=-{{ env_file }}
+# This is a file that the user can use for overrides of the kubelet args as a last resort. Preferably, the user should use
+# the .NodeRegistration.KubeletExtraArgs object in the configuration files instead. KUBELET_EXTRA_ARGS should be sourced from this file.
+EnvironmentFile=-/etc/sysconfig/kubelet
+ExecStart=
```

Comment:
why this line? Isn't the next line enough?

Reply:
It's required to 'clear' the value, otherwise things get appended to what's in the system-global unit file.